src/main/resources/archetype-resources/silo/src/main/resources/application.yml

```diff
@@ -18,7 +30,6 @@ clients:
     common:
         readTimeout: 5000
         connectTimeout: 5000
-        oauthToken: SOMEUUID4
```

Comment:
In Absprache mit @hendrik-rebuy haben wir uns dazu entschlossen, dass die neuen Silos in Zukunft ihre UUIDs aus dem Cache beziehen und zur Laufzeit.